### DISCUSSION TITLE
Make comment policy clearer

### DISCUSSION THREAD

**Discussion by amandasystems** [lukevella/rallly#140] [Ideas]
When using Rallly for booking with my students one of them noted that they did not understand if the comment went straight to the host or was visible to all. This is probably best highlighted by labelling the comment button something like "comment on the poll" or "comment publicly". Or left as it is, but I figured I should propagate the experience.

I am not sure if a "contact author" feature that might send email and/or publish comments only visible to someone with an admin link would also be useful.

**Comment by lukevella** [comment]
Yes, that's true a number of people have wrote to me about this same issue. I think what might help is just adding a badge next to the comment to illustrate that it is public like what you get on facebook and twitter. Something like:

<img width="151" alt="CleanShot 2022-04-27 at 12 16 49@2x" src="https://user-images.githubusercontent.com/676849/165506718-a940b4ed-8af1-43f7-8700-af870062ea5c.png">


The wording and icon will probably be different but that's the general idea. I've added this to the backlog.
